system/data-privacy.rst

```diff
@@ -0,0 +1,137 @@
+Data Privacy
+============
+
+For compliance with GDPR and other data privacy laws,
+you may wish to **permanently delete users from the system**,
+along with **all of their associated tickets**.
+
+.. figure:: /images/system/data-privacy/creating-a-new-deletion-task.png
+   :alt: User deletion dialog in the Data Privacy panel
+   :align: center
+
+   The user deletion dialog lists some of the tickets
+   that will be removed from the system along with the user.
+
+.. note:: 🤔 **Huh? I don’t see the Data Privacy panel...**
+
+   Access to this panel requires ``admin.data_privacy`` permissions
+   (introduced in Zammad 3.5).
+
+   On older systems that have not been updated yet,
+   customers can also be `deleted via the Zammad console`_.
+
+   .. _deleted via the Zammad console: https://docs.zammad.org/en/latest/admin/console/dangerzone-for-experts.html#deleting-customers
+
+Deleting Users
+--------------
+
+.. warning:: 💣 **All deletions are FINAL!**
+
+   Once you click “Delete”, the action cannot be cancelled or undone.
+
+   Any time you delete a user, all their tickets will be deleted, as well.
+   It is not possible to delete a user and still keep their tickets.
+
+.. note:: 🙅 **The following records cannot be deleted:**
+
+  * your own account
+  * the system’s last remaining administrator account
+
+Step 1: Find a user / confirm deletion
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are three ways to access the user deletion dialog:
+
+**from the user’s profile**
+
+   .. figure:: /images/system/data-privacy/delete-user-via-profile.gif
+      :alt: Accessing the user deletion dialog in a user profile
+      :align: center
+
+      Click **Action > Delete**.
+
+**in the “Manage > Users” Admin Panel**
+
+   .. figure:: /images/system/data-privacy/delete-user-via-user-panel.gif
+      :alt: Accessing the user deletion dialog under “Users” in the Admin Panel
+      :align: center
+
+      Use the **⋮ Actions** menu for the target user.
+
+**in the “System > Data Privacy” Admin Panel**
+
+   .. figure:: /images/system/data-privacy/delete-user-via-data-privacy-panel.gif
+      :alt: Accessing the user deletion dialog under “Data Privacy” in the Admin Panel
+      :align: center
+
+      Use the **New Deletion Task** button. Search for users by name or email address.
+
+.. hint:: 👥 **You can delete organizations, too.**
+
+   If the customer you are deleting is the last user in their organization,
+   a **Delete Organization?** option will be displayed in the user deletion dialog:
+
+   .. figure:: /images/system/data-privacy/delete-organization-option.png
+      :alt: Deleting an organization via the user deletion dialog
+      :align: center
+      :width: 50%
+
+   (If you encounter an error when selecting “Yes” here,
+   make sure there are no pending deletion tasks
+   for other customers from this organization.)
```

Comment:
I perosnally think that this text part is not needed.
The reason for this is that Zammad always only provides the organization removal screen if you are removing the last existing user. It does not take in account removal tasks.

I just double checked above.
I'd remove this text? I think you tried to adapt my note that you have to remove all organization members before adding a removal task for the last organization member?

Reply:
Ah, I see what you mean now! How's this?

```suggestion
   (If this option does not appear,
   make sure there are no pending deletion tasks
   for other customers from this organization.)
```